students/piotrowski_stanislaw/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,88 @@
+# Calculator.py
+# piotrsta
+
+
+def help_calc():
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+def add(x, y):
+    print("Result: " + str(x + y))
+
+
+def subtract(x, y):
+    print("Result: " + str(x - y))
+
+
+def multiply(x, y):
+    print("Result: " + str(x * y))
+
+
+def divide(x, y):
+    print("Result: " + str(x / y))
+
+
+def power(x, y):
+    print("Result: " + str(x ** y))
+
+
+def input_operand1():
```

Comment:
możemy to również połączyć w jedną funkcje i wykorzystac zwracanie wielu zmiennych z funkcji ` return a, b, c, d...`
np:
`def f1():`
`...`
`    return a, b`
Użycie funkcji z natychmiastowym rozpakowaniem zmiennych:
`n1, n2 = f1()`

można również rozpakować je później i wyniki trzymać jako tupla:
`liczby = f1()`
`operacja_dodawanie(liczby[0], liczby[1])`
ale to jest temat na później